src/apis/AsyncStorage/index.js

```diff
@@ -40,8 +40,17 @@ class AsyncStorage {
           const key = window.localStorage.key(i);
           keys.push(key);
         }
+
+        if (cb) {
+          cb(null, keys);
+        }
+
         resolve(keys);
```

Comment:
That's my understanding from the documentation (and using AsyncStorage in native).